package.json

```diff
@@ -46,9 +47,11 @@
     "eslint-plugin-jsx-a11y": "6.6.1",
     "eslint-plugin-react": "7.31.11",
     "eslint-plugin-react-hooks": "4.6.0",
+    "husky": "^8.0.3",
     "i18next": "22.4.13",
     "i18next-browser-languagedetector": "7.0.1",
     "i18next-http-backend": "2.2.0",
+    "lint-staged": "^13.2.2",
```

Comment:
We should lock these versions down and remove the caret

Reply:
done. Probably should document somewhere that is our policy.